### PR TITLE
3.0.0 Deprecations

### PR DESCRIPTION
**Pull request by udidol**

### CODE REVIEW

**Review by kobizz — DISMISSED** [review]

File: tests/phpunit/elementor/test-widgets.php [hunk 1]
```diff
@@ -67,7 +69,7 @@ public function test_controlsDefaultData() {
 
 				$error_msg = sprintf( 'Widget: %1$s, Control: %2$s', $widget->get_name(), $control['name'] );
 
-				if ( empty( $control['default'] ) ) {
+				if ( Utils::is_empty( $control['default'] ) ) {
```
Comment (matipojo) [inline tests/phpunit/elementor/test-widgets.php:72]: Please remove the file from PR.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/phpunit/elementor/test-widgets.php (hunk 1 above)
Comment (udidol) [reply tests/phpunit/elementor/test-widgets.php:72]: If I remove this file from the PR, the PR will fail the PHPUnit integration tests

Comment (matipojo) [reply tests/phpunit/elementor/test-widgets.php:72]: Which change cause it to fail?

**Review by matipojo — DISMISSED** [review]

**Review by KingYes — APPROVED** [review]

File: includes/utils.php [hunk 2]
```diff
@@ -30,7 +30,7 @@ class Utils {
 	 * @return bool True if it's a WordPress ajax request, false otherwise.
 	 */
 	public static function is_ajax() {
-		// _deprecated_function( __METHOD__, '2.6.0', `wp_doing_ajax()` );
+		 _deprecated_function( __METHOD__, '2.6.0', `wp_doing_ajax()` );
```
Comment (KingYes) [inline includes/utils.php:33]:
```suggestion
		 _deprecated_function( __METHOD__, '2.6.0', 'wp_doing_ajax()' );
```

**Review by KingYes — CHANGES_REQUESTED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**